Assets/_Functionalities/AmsterdamTimeMachine/Scripts/Ui/AtmInformationList.cs

```diff
@@ -0,0 +1,88 @@
+namespace Netherlands3D.Twin
+{
+    public class AtmInformationList : MonoBehaviour
+    {
+        public class AtmInfo
+        {
+            public string url;
+            public string title;
+            public string startDate;
+            public string endDate;
+            public string textDate;
+            public string thumbnail;
+
+            public override string ToString()
+            {
+                return $"{{url: {url}, title: {title}, startDate: {startDate}, endDate: {endDate}, textDate: {textDate}, thumbnail: {thumbnail}}}";
+            }
+        }
+
+        [SerializeField] private GameObject contentArea;
+        [SerializeField] private AtmInformationListItem photoListItem;
+        [SerializeField] private AtmInformationListItem regularListItem;
+        private const string bagUriTemplate = "https://api.lod.uba.uva.nl/queries/ATM/images-building-nl3d/2/run?bag=http://bag.basisregistraties.overheid.nl/bag/id/pand/{id}";
+        private const string adamLinkTemplate = "https://api.lod.uba.uva.nl/queries/ATM/images-historical-address-nl3d/2/run?address={id}";
+
+        public void LoadFromBagId(string bagId)
+        {
+            QueryParameters parameters = new() {{"id", bagId}};
+
+            PopulateList(new TemplatedUri(bagUriTemplate, parameters));
+        }
+
+        public void LoadFromFeature(FeatureMapping featureMapping)
+        {
+            var adamLinkId = featureMapping.Feature.Properties["id"]?.ToString();
+
+            // check for an id field, otherwise it is not an Amsterdam Time Machine feature
+            if (string.IsNullOrEmpty(adamLinkId)) return;
+            
+            // check if it is a valid Amsterdam Time Machine feature, otherwise we ignore it 
+            if (adamLinkId.StartsWith("https://adamlink.nl/geo/address/") == false) return;
+
+            QueryParameters parameters = new() {{"id", adamLinkId}};
+
+            PopulateList(new TemplatedUri(adamLinkTemplate, parameters));
+        }
+
+        private void PopulateList(Uri url)
+        {
+            contentArea.transform.ClearAllChildren();
+
+            var promise = Resource<List<AtmInfo>>.At(url).Value;
+            promise.Then(CreateListItems);
+            promise.Catch(OnErrorDuringLoading);
+        }
+
+        private void OnErrorDuringLoading(Exception exception)
+        {
+            var error = exception as Error;
+            
+            Debug.LogError(
+                $"An issue occurred while loading data from ATM ({error?.Request.Url}):\n{exception.Message}"
+            );
+        }
+
+        private void CreateListItems(List<AtmInfo> atmInformationObjects)
+        {
+            foreach (var atmInfo in atmInformationObjects)
+            {
+                CreateListItem(atmInfo);
+            }
+        }
+
+        private void CreateListItem(AtmInfo atmInfo)
+        {
+            var listItemPrefab = string.IsNullOrEmpty(atmInfo.thumbnail) ? regularListItem : photoListItem;
```

Comment:
Valt mij op als ik dubbel click op een pand in de build dat de lijst 2x word gedaan en dus 2x zoveel items onder de content gehangen worden

Reply:
Goeie, daar moet dan naar gekeken worden. Kunnen we dat als subtaak aanmaken aan de story en deze alvast mergen om merge conflicts te voorkomen?